apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/internal/MapJsonReader.kt

```diff
@@ -64,20 +69,27 @@ class MapJsonReader(val root: Map<String, Any?>) : JsonReader {
     val map = nextValue()
     check(map is Map<*, *>)
     push(OrderedMap(map.entries.map { Entry(it.key as String, it.value) }))
+
+    nameIndexStackSize++
+    check(nameIndexStackSize < 33) {
```

Comment:
where this comes from?

Reply:
That's from Moshi: https://github.com/apollographql/apollo-android/blob/caf408b6cd0f0d39547675310b1aa4f1ccebecc2/apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/internal/json/BufferedSourceJsonReader.kt#L50

We don't support more than 32 levels of nesting